portfolio/src/main/webapp/script.js

```diff
@@ -13,16 +13,56 @@
 // limitations under the License.
 
 /**
- * Adds a random greeting to the page.
+ * Adds a random picture with its description to the page.
  */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
 
-  // Pick a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
+  /** !Array<Objects> */
+  /* @type {{imgSrc: string, description: string }}  */
```

Comment:
```suggestion
  /** @type {{imgSrc: string, description: string, altText: string }}  */
```
JSDoc starts with "/**" and ends with "*/"